### PR TITLE
041 lazy load infinite scroll

### PR DESCRIPTION
**Pull request by Guin-**

### CODE REVIEW

File: frontend/app/components/catList.jsx [hunk 1]
```diff
@@ -1,27 +1,91 @@
...
+    }, []).map((rowContent, index) => {
+      return <div className="row" key={index}>{rowContent}</div>
+    })
+
     return (
       <Grid>
-        <Row>
-          {catItems}
-        </Row>
+        <Media query="(max-width: 1024px)">
```
Comment (Guin-) [inline frontend/app/components/catList.jsx]: Trying to figure out if there is a way i could refactor this or use a different library, but for now it gets the job done.

File: frontend/app/actions/persistZip.jsx [hunk 2]
```diff
@@ -0,0 +1,5 @@
+export const PERSIST_ZIP = 'PERSIST_ZIP'
```
Comment (galarant) [inline frontend/app/actions/persistZip.jsx]: This needs to clear if the user navigates back to the main page and searches a different zip. At the moment it just persists the first zip you typed for the entire session.

File: frontend/app/components/catList.jsx [hunk 3]
```diff
@@ -1,27 +1,91 @@
...
+            <CatCard cat={cat} key={cat.id} />
+        )
     })
+    const rows = mobileRows.reduce((row, element, index)=>{
+        index % groupSize === 0 && row.push([]);
+        row[row.length - 1].push(element);
+        return row
+    }, []).map((rowContent, index) => {
+      return <div className="row" key={index}>{rowContent}</div>
+    })
```
Comment (galarant) [inline frontend/app/components/catList.jsx]: this is not very readable. I understand the elegant feeling of packing a ton of logic into as few lines as possible, but I think you overdid it here. if you don't want to refactor for readability that's fine but at least add an explanatory comment

File: frontend/app/components/catList.jsx (hunk 1 above)
Comment (galarant) [reply frontend/app/components/catList.jsx]: I think it looks good!

**Pull request merged**